govuk-one-login/authentication-api
# BAU: Upgrade terraform and providers

## What

- Update terraform to 1.9.8
- Update provider versions

We haven't bumped terraform for a while.

## How to review

1. Deploy `dev` from `main`
2. Deploy `dev` from this branch
3. Observe that no errors / no obsurd changes occur
